packages/airnode-utilities/src/logging.ts

```diff
@@ -0,0 +1,36 @@
+import { TextEncoder } from 'util';
+
+type LogType = 'DEBUG' | 'INFO' | 'WARN' | 'ERROR';
```

Comment:
Maybe this is a bit clearer than "type"?

```suggestion
type LogLevel = 'DEBUG' | 'INFO' | 'WARN' | 'ERROR';
```